app/controllers/api/v1/ecf_participants_controller.rb

```diff
@@ -14,21 +14,52 @@ class ECFParticipantsController < Api::ApiController
       def index
         respond_to do |format|
           format.json do
-            participant_hash = ParticipantFromInductionRecordSerializer.new(paginate(induction_records)).serializable_hash
+            participant_hash = ParticipantSerializer.new(paginate(participant_profiles), params: { lead_provider: }).serializable_hash
             render json: participant_hash.to_json
           end
 
           format.csv do
-            participant_hash = ParticipantFromInductionRecordSerializer.new(induction_records).serializable_hash
+            participant_hash = ParticipantSerializer.new(participant_profiles, params: { lead_provider: }).serializable_hash
             render body: to_csv(participant_hash)
           end
         end
       end
 
     private
 
-      def induction_records
-        @induction_records ||= ECFParticipants::Index.new(cpd_lead_provider: current_user, params:).induction_records
+      def participant_profiles
+        join = InductionRecord
+                 .select("induction_records.participant_profile_id, induction_records.training_status, induction_records.updated_at, induction_records.created_at, ROW_NUMBER() OVER (PARTITION BY participant_profiles.participant_identity_id ORDER BY induction_records.created_at DESC) AS training_status_precedence")
```

Comment:
`training_status_precedence`, this column has nothing to do with `training_status` can it be given a more appropriate name?